### DISCUSSION TITLE
Branch limitations.

### DISCUSSION THREAD

**Discussion by rohit2g** [community/community#144199] [Repositories]
### Select Topic Area

General

### Body

Is there a limitation or recommendation on maximum number of branches a given repository is permitted?

**Comment by Subhadipgayen2006** [comment]
Maybe

**Comment by siba-9439** [comment]
Hi @rohit2g,

First of all there is no limitation to create branches of your repository because branches are very small thing and lightweight reference, you can create n number of branches according to your requirement. There is only one cons can be arises that is your performance may be degraded while doing some operations in CLI like git fetch, git log, git push etc. ,if you have an extremely large number of branches.

If it is helpful to you, please choose the resolution as an answer. For more info you can checkout my profile.
